clients/includes/further-reading.inc

```diff
@@ -0,0 +1,4 @@
+* Use :ref:`plugins <plugins>` to customize the way HTTP requests are sent and responses
+  processed.
```

Comment:
we could mention that the plain client is configured to not follow redirects for example.